tslint.json

```diff
@@ -3,16 +3,17 @@
   "rules": {
     "ban": [true,
       ["Object", "assign", "use TS2.1 object spread { ...a, ...b }"],
+      ["Object", "getOwnPropertyNames", "use Object.keys()"],
```

Comment:
it picks up non-enumerable properties too. I've never had a need to do that, it's easier to stick with just object.keys https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/getOwnPropertyNames